### DISCUSSION TITLE
Feature suggestion: use our own form for account requests

### DISCUSSION THREAD

**Discussion by damithc** [TEAMMATES/teammates#11786] [Ideas]
Current: An embedded Google form is used for collecting new account requests https://teammatesv4.appspot.com/web/front/request

Suggestion: use our own form instead

Pros: 
* Gives a better impression to potential new users (using Google forms seem like we are short of resources to create our own)
* We can do more validity checks on input data e.g., reject students requesting for accounts, reject gmail/hotmail email addresses
* Makes it easier for admin to create new accounts

What do you guys think?

**Comment by samuelfangjw** [comment]
I think this is a good idea!

**Comment by wkurniawan07** [comment]
This is not just a good idea. This is an essential feature long overdue.

**Comment by wkurniawan07** [comment]
Close as this is eesentially duplicate of https://github.com/TEAMMATES/teammates/wiki/Account-Request-Form-Upgrade
